scripts/dev/misc_checks.py

```diff
@@ -189,6 +189,7 @@ def check_spelling(args: argparse.Namespace) -> Optional[bool]:
         'mouseMove',
         'mouseDClick',
         'keySequence',
+        'waitSignal',
```

Comment:
This is only for the Qt methods for which no snake case variant exists (see my comment below), so it shouldn't be in here.